.travis.yml

```diff
@@ -0,0 +1,11 @@
+sudo: false
+cache: bundler
+language: ruby
+rvm:
+  - 2.4.6
+  - 2.5.3
+env:
+  - LD_LIBRARY_PATH=spec/ext
+before_install:
+  - gem install bundler -v 1.14.4
```

Comment:
Why this specific bundler version?

Reply:
No reason, I just copied this from one of our other gems to get it working